tests/e2e/specs/out-of-game/home.spec.js

```diff
@@ -1,5 +1,12 @@
 import { assertSnackbarError, Card } from '../../support/helpers';
-import { myUser, opponentOne, opponentTwo, playerOne, playerTwo } from '../../fixtures/userFixtures';
+import {
+  myUser,
+  opponentOne,
+  opponentTwo,
+  playerOne,
+  playerTwo,
+  playerThree,
```

Comment:
Looks like this variable isn't used; let's remove the import. This is the cause of the linting error in CI
```suggestion
```